### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 12

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#284] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152319412-28aff93e-c753-487f-bb06-621ebeca234d.png)

**Comment by Elia-Belli** [comment]
$R \cap R^{-1} \subseteq D_A$ vuol dire che gli unici elementi in comune tra $R$ e $R^{-1}$ sono le coppie del tipo $(a,a)$.

Per assurdo supponiamo che $R$ non sia antisimmetrica:
Quindi possono esistere $aRb$ e $bRa$, con $a\not= b$, allora sicuramente $bR^{-1}a$ e $aR^{-1}b$, quindi l'intersezione contiene anche $(a,b)$ e $(b,a)$ che non sono elementi $\in D_A$, è violata quindi l'ipotesi.

Concludiamo che se $R \cap R^{-1} \subseteq D_A$ allora $R$ è necessariamente antisimmetrica. $Q.E.D.$
